c/cjson/cjson.go

```diff
@@ -26,128 +26,650 @@ const (
 	LLGoPackage = "link: $(pkg-config --libs libcjson); -lcjson"
 )
 
+type JSON_bool c.Int
```

Comment:
use `Bool` instead of `JSON_bool`

Reply:
> use `Bool` instead of `JSON_bool`

Will conflict with this function name
```go
//go:linkname Bool C.cJSON_CreateBool
func Bool(boolean c.Int) *JSON
```